.vale/styles/Infrahub/sentence-case.yml

```diff
@@ -15,40 +15,50 @@ exceptions:
   - Ansible
   - API
   - Attributes
+  - AWS S3
   - Beta
   - BGP
+  - CA certificates
   - CI
   - Codespaces
   - CoreArtifactTarget
   - CoreRepository
+  - CoreGroup
   - Docker Compose
   - E2E
   - Generics
   - Git
   - GitHub
+  - GitHub Codespaces
   - GitLab
   - GitPod
   - GraphQL
   - GraphQLQuery
   - include_in_menu & menu_placement
   - Helm
+  - High Availability
```

Comment:
Don't think we need this kind of casing.

Reply:
you would say `Infrahub High availability deployment` instead of `Infrahub High Availability` ?